vulcanai2/plotters/visualization.py

```diff
@@ -41,12 +79,6 @@ def display_record(record=None, load_path=None, interactive=True):
 
     """
     title = 'Training curve'
-    if load_path is not None:
-        with open(load_path) as in_file:
-            record = pickle.load(in_file)
-        title = 'Training curve for model: {}'.format(
-            os.path.basename(load_path))
```

Comment:
It would be nice to have a separate load function to replace this, for those who wish to view their images that way.

Reply:
we've created issue #109 to flush this out a little more